projects/tcpdump/build.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -eu
+#
+# Copyright (C) 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+#Build targets for libfuzzer
+if [[ $CONFIG = "libfuzzer" ]]
+then
+    export CC="clang"
+    export CFLAGS="-g -fsanitize=fuzzer-no-link,address,bounds,integer,undefined,null,float-divide-by-zero"
+    export TARGET_CFLAGS="-g -fsanitize=fuzzer,address,bounds,integer,undefined,null,float-divide-by-zero"
+    export LINK_FLAGS="$TARGET_CFLAGS -I/libpcap -I/tcpdump ../load_sydr_packet.c -o ../fuzzer_packet\
+                       /libpcap/libpcap.a /tcpdump/libnetdissect.a"
+fi
+
+if [[ $CONFIG = "afl" ]]
+then
+    cd /libpcap && make clean
+    cd /tcpdump && make clean
+    export CC="afl-clang-fast"
+    export CFLAGS="-g -fsanitize=address,bounds,integer,undefined,null,float-divide-by-zero"
+    export TARGET_CFLAGS="-g -fsanitize=fuzzer,address,bounds,integer,undefined,null,float-divide-by-zero"
```

Comment:
Do we need `TARGET_CFLAGS` be separate from `LINK_FLAGS`?

Reply:
removed